src/store.js

```diff
@@ -1,3 +1,4 @@
 import { createStore } from 'redux';
+import reducer from './reducers';
 
-export default createStore(() => ({}));
+export default createStore(reducer);
```

Comment:
```suggestion
export default createStore(
  reducer,
  window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__(),
);
```
Highly recommend bringing in the dev tools for easier inspection!